inc/package.class.php

```diff
@@ -580,6 +580,11 @@ protected function sendFile() {
 
       exit(0);
    }
+
+   /**
+    * Deletes the entities 
```

Comment:
It does not delete an entity. It depetes all packages having a relation to the entity given as argument. This method is called when something purges an entity in GLPI.